pykafka/balancedconsumer.py

```diff
@@ -553,6 +566,22 @@ def _topics_changed(self, topics):
         log.debug("Rebalance triggered by topic change")
         self._rebalance()
 
+    def _zookeeper_state_changed(self, state):
+        if state in (KazooState.LOST, KazooState.SUSPENDED):
```

Comment:
I don't dig too much about LOST and SUSPENDED. From the doc of kazoo, ephemeral nodes may or may not be lost in SUSPENDED state.
I don't drop the partitions unilaterally. I have checked if the consumer id is still there. Only if not, i will drop the partitions.